client/src/test/java/com/paypal/selion/TravisCICapabilityBuilder.java

```diff
@@ -12,39 +12,39 @@
 |  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for  |
```

Comment:
Note: GeckoCapabilitiesBuilder was not renamed. it was deleted... Git issue here ... The diff for this file is not right.. TestCapabilityBuilder was renamed to TravisCICapabilityBuilder